tests/model_selection/dask_searchcv/test_model_selection.py

```diff
@@ -891,3 +898,56 @@ def test_mock_with_fit_param_raises():
 
     with pytest.raises(ValueError):
         clf.fit(X, y)
+
+
+@pytest.mark.parametrize(
+    "cv_ind,exp_acc",
+    [
+        ([("train", "test")], 1 / (1 + 99)),
+        ([("test", "train")], 100 / (100 + 200)),
+        (
+            [("train", "test"), ("test", "train")],
+            (1 / (1 + 99) + 100 / (100 + 200)) / 2,
+        ),
+        (
+            [("test", "train"), ("train", "test")],
+            (100 / (100 + 200) + 1 / (1 + 99)) / 2,
+        ),
+    ],
+)
+def test_sample_weight_in_metrics(cv_ind, exp_acc):
```

Comment:
**NOTE**: If this test were used in the current master of *dask_ml*, it would fail because the metric returned by all tests is `0.5`.

`0.5` **is *definitely* incorrect** as it disregards `sample_weight` entirely.